src/lib/child_processes/child_processes.ml

```diff
@@ -275,8 +276,8 @@ let start_custom :
       (List.filter_opt
          [ Unix.getenv @@ "CODA_" ^ String.uppercase name ^ "_PATH"
          ; relative_to_root
-         ; Some (Filename.dirname coda_binary_path ^/ name)
-         ; Some ("coda-" ^ name) ])
+         ; Some (Filename.dirname mina_binary_path ^/ name)
+         ; Some ("mina-" ^ name) ])
```

Comment:
Will this break anything?

Reply:
The binaries we distribute lately are named `mina-`, like `mina-validate-keypair`. So I believe this change is a strict improvement.